README.md

```diff
@@ -22,43 +22,31 @@ It's easy to use flannel with Kubernetes. Flannel can use the Kubernetes API as
 
 ### Adding flannel
 
-Flannel can be added to any existing Kubernetres cluster. It's simplest to add `flannel` before any pods using the pod network have been started.
+Flannel can be added to any existing Kubernetes cluster. It's simplest to add `flannel` before any pods using the pod network have been started.
 
-A great way to create a new Kubernetes cluster is with [kubeadm][kubeadm] which helpfully points out the need to define *a pod network CIDR* for example by passing in `--pod-network-cidr 10.244.0.0/16` when running `kubeadm init`. This pod network CIDR is the key prerequisite for using flannel with the kube subnet manager. If you're running flannel through some other means, then the pod CIDR might be set automatically for you or you can set it manually by passing [--pod-cidr][pod-cidr] to each of your kubelets.
+While several Kubernetes deployment tools use flannel by default, the quickest way to get up and running is to use the open source [Tectonic Installer][tectonic-installer], which provides good defaults, and enables install automation.
 
-It's also possible to set it per-node using `kubectl patch`:
-```bash
-kubectl patch node minikube -p '{"spec":{"podCIDR":"10.33.12.0/24"}}'
-```
+For information on deploying flannel manually, using the (currently alpha) Kubernetes installer toolkit kubeadm, see [Installing Kubernetes on Linux with kubeadm][installing-with-kubeadm].
 
-Once the `podCIDR` is set, add `flannel` to the cluster by applying the manifest file when using `kubectl apply`:
-```bash
-kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
-```
+### Using flannel
 
-If you're using RBAC, also apply kube-flannel-rbac.yaml:
-```bash
-kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel-rbac.yml
-```
-
-Although not required, it's highly recommended that you use CNI to start your pods. This requires your kubelets to be started with `--network-plugin=cni`. This is the default for clusters created by `kubeadm`.
+(This section should describe, in broad terms, all the reasons you would want to have flannel up and running. Supporting docs give details.)
```

Comment:
@tomdee @joshix please note - I removed much of the kubeadm material, but I'm really not clear on whether the stuff I left is more generic or not. I think this section should should fulfill Brandon's comment, and "show them indepth ways on how to manage it". The specifics can live in running.md. This section should simply tell people what you can do with flannel, once it's up and running / why it's great. I think pinging pod to pod is one of these things. what are the others?